examples/testbed_simple/android/app/build.gradle

```diff
@@ -178,6 +178,9 @@ android {
 dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
     implementation "com.facebook.react:react-native:+"  // From node_modules
+    implementation 'com.google.firebase:firebase-appindexing:19.1.0'
+    implementation 'com.google.android.gms:play-services-ads-identifier:17.0.0'
```

Comment:
I believe version 17 requires extra configuration steps , are you sure this works on Android? See warning `Google Play Services version 17+` for more information (https://help.branch.io/developers-hub/docs/android-basic-integration) If not, it's safe to return to v16+.

Reply:
I just added this since I was having trouble testing at first. I guess I misread 16+ as `>= 16` and assumed this was also correct and more current. I can remove this or correct it. It's probably more realistic to add these things to the examples.

Reply:
Interesting, running the test app worked for me too. Perhaps the extra configuration is needed only when importing `com.google.android.gms:play-services-ads` not `com.google.android.gms:play-services-ads-identifier` in which case I need to make a DOCS ticket.. But yeah, 16+ means the highest minor/patch version of version 16.